build/media_source/system/js/fields/passwordcaps.es6.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @copyright   (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// Detect Caps Lock status
+function checkCapsLock(event) {
+  const capsLockOn = event.getModifierState && event.getModifierState('CapsLock');
+  const target = event.target;
```

Comment:
```suggestion
  const { target } = event;
```

First Eslint warning